gcc-4.8/post-link.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+if [ "$(uname)" != "Darwin" ]; then
+
+    # Locate the system's C-runtime object files and link them into the gcc 
+    #  build so they are automatically on the gcc search path.
+    # (The location of these files varies from one system to the next.)
+    C_RUNTIME_OBJ_FILES="crt1.o crti.o crtn.o"
+    SYSTEM_GCC=/usr/bin/gcc
+    
+    DUMMY_PROGRAM="int main(){return 0;}"
+    GCC_CMDS=`echo "$DUMMY_PROGRAM" | $SYSTEM_GCC -v -xc - 2>&1`
```

Comment:
There may be a simpler way for me to get the locations of `crt1.o`, etc.  I'll revise this patch on Monday.